### PR TITLE
add autoComplete and placeholder 

### PR DESCRIPTION
**Pull request by marcelmokos**

### CODE REVIEW

**Comment by mattiaerre** [comment]
Thank you @marcelmokos for you PR. 
I will review it and merge it ASAP.
Then I will publish a new version of the component.

**Comment by marcelmokos** [comment]
I have done some cleaning, also I have added readme for this change.

**Comment by mattiaerre** [comment]
this is really good thanks @marcelmokos maybe you would like to add the **id** as well as per #1 ?

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ And use it inside your container:
 ```javascript
 <InputBoxDoneTyping
   inputClassName="form-control"
+  inputPlaceholde="Input text here"
```
Comment (mattiaerre) [inline README.md]: do you mind to change the message into something like "Start typing ..." ?

File: demo/index.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const App = (props) => {
       <div className="row">
         <InputBoxDoneTyping
           inputClassName="form-control"
+          inputPlaceholde="Input text here"
```
Comment (mattiaerre) [inline demo/index.js]: see comment above

File: src/InputBoxDoneTyping.js [hunk 3]
```diff
@@ -30,21 +30,31 @@ const InputBoxDoneTyping = (props) => {
...
+       placeholder={props.inputPlaceholder}
+       defaultValue={props.inputDefaultValue}
+       onChange={handleOnChange}
+       onKeyUp={handleOnKeyUp}
+       onKeyDown={handleOnKeyDown}
+    />
   );
 }
 
+InputBoxDoneTyping.defaultProps = {
```
Comment (mattiaerre) [inline src/InputBoxDoneTyping.js:46]: I like that 👍

**Comment by marcelmokos** [comment]
I have added inputId and .editorconfig, sorry for that many commints next time I will squash commits together

**Comment by mattiaerre** [comment]
LGTM 👍 shall I merge it? @marcelmokos

**Comment by mattiaerre** [comment]
no problem at all for the commits; I will squash them while merging 😉

**Comment by marcelmokos** [comment]
Sure you can merge it. Thanks

**Comment by marcelmokos** [comment]
Can you update version on npm.  
https://www.npmjs.com/package/react-input-box-done-typing

**Comment by mattiaerre** [comment]
hi @marcelmokos just published the **v1.1.0** release; thank you so much for your help and any other feedback let me know

https://www.npmjs.com/package/react-input-box-done-typing

**Pull request merged**